# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../adm8211/2.6.20.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- adm8211/wep.c.vanilla	2007-03-30 09:36:24.446241003 +0200
+++ adm8211/wep.c	2007-03-30 09:44:01.860307546 +0200
@@ -22,6 +22,8 @@
 #include <linux/crc32.h>
 #include <asm/string.h>
 #include <asm/scatterlist.h>
+#include <asm/page.h>
+#include <linux/mm.h>
 
 #ifndef CONFIG_CRYPTO
 #error CONFIG_CRYPTO required to build
